src/helpers.js

```diff
@@ -1,3 +1,6 @@
+import { isFunction } from "lodash"
```

Comment:
This will add lodash to a dependency for the client.  There is an issue open now questioning the need for lodash at all. (#24).  You might look at pulling the utility out of lodash and into the project so the extra dependency can be avoided.